plugin/trino-hudi/pom.xml

```diff
@@ -374,6 +374,13 @@
             <scope>test</scope>
         </dependency>
 
+        <dependency>
+            <groupId>org.mockito</groupId>
+            <artifactId>mockito-core</artifactId>
```

Comment:
We basically don't use mockito in tests. Please fix `BaseHudiConnectorTest` or `TestHudiSparkCompatibility` (or both).

Reply:
The test in `trino-hudi` is ready. I have created a simple copy of the already existing test table `stock_ticks_cow` in which the Hudi timeline has `replacecommit` actions, that is present in the patchset. I have confirmed that without the code change the test is also failing exactly with the above mentioned exception. Please let me know what do you think about this solution/direction.

Still need some time to familiarize myself with the product tests and add a test case there as well for this fix.

Reply:
I extended `TestHudiSparkCompatibility` in the latest patchset, verifying that it also fails with the `No scheme for file system location:` exception above. Modified the setup method as well to have a default value for `S3_BUCKET` if that is not specified externally. Please let me know what do you think about this solution in overall.

Signing of the CLA is underway, I will send it in the upcoming days.

Reply:
@ebyhr Are you OK with the proposed changes? Thanks for taking a look again at this!